ops/services/app_functions/report_stream_batched_publisher/infra/_data.tf

```diff
@@ -15,6 +15,11 @@ data "azurerm_key_vault_secret" "datahub_api_key" {
   key_vault_id = data.azurerm_key_vault.sr_global.id
 }
 
+data "azurerm_key_vault_secret" "report_stream_callback_token" {
```

Comment:
how hard is it to rename this in the future, I know it will need a code change? It should be called `simple_report_callback_token` since its a token for SR api to report Report stream warnings/errors

Reply:
What do you want the new secret to be called? I can do `simple-report-callback-token-ENV` but that seems like it leaves out ReportStream, so it's not immediately obvious what this value is for when browsing through Key Vault.

Reply:
yeah I think `simple-report-callback-token-ENV` would work, we only make the call back to SimpleReport api, and even though the contents are report-stream related, the Api token has nothing to do with report-stream.

Reply:
New commit pushed